code/game/objects/items/tools.dm

```diff
@@ -457,6 +457,15 @@
 		flamethrower_screwdriver(I, user)
 	else if(istype(I, /obj/item/stack/rods))
 		flamethrower_rods(I, user)
+	else if(istype(I, /obj/item/reagent_containers) && is_open_container())

```

Comment:
@optimumtact Shouldn't this have been I.is_open_container()???????

Reply:
uhhh

Reply:
yes

Reply:
to be fair all this would do is allow you to use pills to fuel it